MarineSABRES_SES_Shiny/scripts/kb_audit/audit_wp5_mechanisms.py

```
@@ -0,0 +1,137 @@
+        for service, vals in pos.items():
+            if not all(k in vals for k in ("low", "central", "high", "unit", "method")):
+                issues.append(("FAIL", f"valuation_unit_values.posidonia_oceanica.{service}: missing low/central/high/unit/method"))
+            elif not (vals["low"] <= vals["central"] <= vals["high"]):
+                issues.append(("FAIL", f"valuation_unit_values.posidonia_oceanica.{service}: bands not ordered low <= central <= high"))
+
+    # Report
+    fails = [i for i in issues if i[0] == "FAIL"]
+    warns = [i for i in issues if i[0] == "WARN"]
+
+    n_mechs = sum(len(da.get("mechanisms", [])) for da in das.values())
+    print(f"=== WP5 mechanism KB audit ===")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)